### PR TITLE
v1.41 Attendance bug fixes and UGDG Language corrected 

### LINKED ISSUE #256: [Attendance] Repeated lines for invalid date in replaceAtten command
image

### LINKED ISSUE #266: [Attendance] viewAttendanceDate command
Message usage states paramter as d/date. should be d/dd-mm-yyyy.

### LINKED ISSUE #267: [Attendance] Command parameters
Use INDEX and caps for all parameters.

### LINKED ISSUE #268: [Attendance] viewAttendace wrong argument
viewAttenPerson abc gives "For input string: "abc""

### PR DESCRIPTION
**Pull request by meisbokai**
Resolves the following Issues:
[Attendance] viewAttendace wrong argument 
Resolves #268 
[Attendance] Command parameters 
Resolves #267 
[Attendance] viewAttendanceDate command 
Resolves #266 
[Attendance] Repeated lines for invalid date in replaceAtten command 
Resolves #256 

UGDG has corrected grammer and spelling

### CODE REVIEW

**Comment by micklwl** [comment]
Resolves xxx. in your comment.

File: src/seedu/addressbook/commands/attendance/UpdateAttendanceCommand.java [hunk 1]
```diff
@@ -19,7 +19,7 @@
     public static final String COMMAND_WORD = "attendance";
     public static final String MESSAGE_USAGE = COMMAND_WORD + ":\n"
             + "Updates the attendance of a student. \n"
-            + "Parameters: indexOfPerson d/dd-mm-yyyy att/attendance \n"
+            + "Parameters: INDEX d/dd-MM-yyyy att/ISPRESENT \n"
             + "\tExample: " + COMMAND_WORD + " " + "1 d/29-09-2018 att/1 \n"
```
Comment (0WN463) [inline src/seedu/addressbook/commands/attendance/UpdateAttendanceCommand.java]: Intriguing, why is only the month capitalized lawl?

Comment (micklwl) [reply src/seedu/addressbook/commands/attendance/UpdateAttendanceCommand.java]: I think it should be DATE instead of the format

**Comment by 0WN463** [comment]
Also, merge conflict

File: src/seedu/addressbook/commands/attendance/UpdateAttendanceCommand.java (hunk 1 above)
Comment (meisbokai) [reply src/seedu/addressbook/commands/attendance/UpdateAttendanceCommand.java]: Mick, u literally told me to change it from date to the format.

Tek In, mm means minutes. MM is month

Comment (0WN463) [reply src/seedu/addressbook/commands/attendance/UpdateAttendanceCommand.java]: Its just odd that the rest uppercase everything, while yours is only the month

Comment (meisbokai) [reply src/seedu/addressbook/commands/attendance/UpdateAttendanceCommand.java]: i'll change all to caps then

Comment (micklwl) [reply src/seedu/addressbook/commands/attendance/UpdateAttendanceCommand.java]: Sorry about that, I was comparing with your other commands. However, now that I look through everyone's commands, for PARAMETERS, they are words and not format related

**Pull request merged**